src/popup/router/components/NodeConnectionStatus.vue

```diff
@@ -49,34 +46,19 @@ export default {
     position: sticky;
   }
 
-  bottom: 48px;
-  bottom: calc(48px + env(safe-area-inset-bottom));
   left: 0;
   right: 0;
-  background: variables.$color-blue;
+  background: variables.$color-bg-3-new;
+  padding: 4px 0;
   color: variables.$color-white;
   text-align: center;
+  bottom: env(safe-area-inset-bottom);
```

Comment:
In my opinion this component should not position itself. Parent components should take care on this.

Reply:
I did not introduce it in that PR, and i don't think we have time for now to change the approach.